test/screenshot/spec/mdc-chips/classes/choice.html

```diff
@@ -44,33 +44,33 @@
       <div class="test-layout">
         <div class="test-cell test-cell--chip">
           <div class="mdc-chip-set mdc-chip-set--choice">
-            <button class="mdc-chip">
+            <button class="mdc-chip" role="radio" aria-checked="false">
               <span class="mdc-chip__text">Chip One</span>
             </button>
-            <button class="mdc-chip">
+            <button class="mdc-chip" role="radio" aria-checked="false">
               <span class="mdc-chip__text">Chip Two</span>
             </button>
-            <button class="mdc-chip">
+            <button class="mdc-chip" role="radio" aria-checked="false">
               <span class="mdc-chip__text">Chip Three</span>
             </button>
-            <button class="mdc-chip">
+            <button class="mdc-chip" role="radio" aria-checked="false">
               <span class="mdc-chip__text">Chip Four</span>
             </button>
           </div>
         </div>
 
         <div class="test-cell test-cell--chip">
           <div class="mdc-chip-set mdc-chip-set--choice">
-            <button class="mdc-chip mdc-chip--selected">
+            <button class="mdc-chip mdc-chip--selected" role="radio" aria-checked="true">
```

Comment:
If this is a radio type selection update this demo to have single selection.

Reply:
Choice chips are single selection by default. I don't think there's any change to be done for that.

Reply:
Right. All radio buttons have `aria-checked="true"` which is not intended for radio group. Am I missing something?

Reply:
Maybe I'm missing something. Can you make a commit or open a PR with the change you want? I'm not following.